utils/sl/env_modules.py

```diff
@@ -110,8 +110,8 @@ def _init_module_func(self, default_func):
     def _init_mpi_module(self):
         """init mpi module function"""
         try:
-            subprocess.check_call(['sh', '-l', '-c', 'module -V'], \
-                                  stdout=DEVNULL, stderr=DEVNULL)  #nosec
```

Comment:
Don't we still need to specify #nosec in these cases? Or is Bandit satisfied?

Reply:
please see the new commit.